samples/client/petstore/java/okhttp-gson-parcelableModel/src/main/java/org/openapitools/client/JSON.java

```diff
@@ -64,7 +64,8 @@ public Class getClassForElement(JsonElement readElement) {
             }
           })
         ;
-        return fireBuilder.createGsonBuilder();
+        GsonBuilder builder = fireBuilder.createGsonBuilder();
```

Comment:
did you forget to put the `{{#disableHtmlEscaping}}` ?

Reply:
No, I did not forget, I chose to always declare the variable and conditionally set `disableHtmlEscaping`:

```
{{#disableHtmlEscaping}}
builder.disableHtmlEscaping();
{{/disableHtmlEscaping}}
```

This keeps the complexity of the mustache code low, and makes it easy to accommodate more options in the future. For example:

```
{{#disableHtmlEscaping}}
builder.disableHtmlEscaping();
{{/disableHtmlEscaping}}
{{#someNewOption}}
builder.someNewOption();
{{/someNewOption}}
```

I can modify the template to handle the "no options" situation to avoid declaring the variable in generated Java-classes, if that is preferable?

Reply:
It's my bad, since the path to the file was truncated by github UI, I did not realize it was a generated sample. Makes total sense! Keep it that way :)